### PR TITLE
Loadbalancer instructions

### PR DESCRIPTION
**Pull request by DeRuina**
## Proposed changes

Added clearer instructions to loadbalancer 

Briefly describe the changes you've made here. Remember to add a link to the [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) of your branch.

https://csc-guide-preview.2.rahtiapp.fi/origin/loadbalancer_selector/cloud/rahti2/networking/#using-loadbalancer-service-type-with-dedicated-ips

https://csc-guide-preview.2.rahtiapp.fi/origin/loadbalancer_selector/cloud/rahti2/concepts/#service

## Checklist before requesting a review

- [x] I have followed the instructions in the [Contributing](https://github.com/CSCfi/csc-user-guide/blob/master/CONTRIBUTING.md) and [Styleguide](https://github.com/CSCfi/csc-user-guide/blob/master/STYLEGUIDE.md) documents.
- [x] My pull request passes the automated tests. If not, visit [pull requests at Travis CI](https://app.travis-ci.com/github/CSCfi/csc-user-guide/pull_requests) and have a look at the job log.
- [x] I have included a link to the appropriate [preview page](https://csc-guide-preview.2.rahtiapp.fi/origin/) (select your branch from the list).

### CODE REVIEW

File: docs/cloud/rahti2/networking.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ Ensure that the service type is set to `LoadBalancer`, and that the `allocateLoa
 
 Additionally, the port field in the service definition (e.g., `33306` in the previous example) must be within the range of `30000-35000`.
 
+- **How to retrieve the selector**: on your CLI run `oc describe pod <pod-name> -n <namespace>`. After running the oc command, you will see an output that includes a section labeled `Labels`, Copy any of labels and paste in the `yaml` file under `selector`. **Make sure to follow the `yaml` syntax and change `=` to `:`**.
+
```
Comment (lvarin) [inline docs/cloud/rahti2/networking.md:141]: Should you add a snippet of the output?

```yaml
Name:             nginx-okd-c75895d47-99jqb
Namespace:        test-ingress-ip
Priority:         0
Service Account:  default
Node:             2-rcr6j-worker-0-qgtjh/192.168.0.16
Start Time:       Thu, 24 Oct 2024 08:54:07 +0300
Labels:           app=nginx-okd
                  deployment=nginx-okd
                  pod-template-hash=c75895d47
(...)
```

File: docs/cloud/rahti2/networking.md [hunk 2]
```diff
@@ -137,6 +137,10 @@ Ensure that the service type is set to `LoadBalancer`, and that the `allocateLoa
...
+- **How to make sure your service is pointing to the right pod**: On your CLI run `oc get endpoints <service-name> -n <namespace>`. You should see the name of the Service and the IP addresses and ports of the Pods that are currently targeted by the Service. 
+
```
Comment (lvarin) [inline docs/cloud/rahti2/networking.md:143]: Put a small explanation on how to do it in webui and two screen shots?

File: docs/cloud/rahti2/networking.md (hunk 1 above)
Comment (DeRuina) [reply docs/cloud/rahti2/networking.md:183]: Added as requested

File: docs/cloud/rahti2/networking.md (hunk 2 above)
Comment (DeRuina) [reply docs/cloud/rahti2/networking.md:184]: Added as requested

**Comment by DeRuina** [comment]
can @lvarin review the new changes?

**Review by lvarin — APPROVED** [review]
LGTM

**Pull request merged**